text/draw_text_shaper.cpp

```diff
@@ -49,11 +50,31 @@ class AdapterBuilder : public TextBlob::RunHandler {
   void commitRunBuffer(TextBlob::RunInfo& info) override
   {
     if (info.clusters && info.glyphCount > 0) {
-      float advanceX = 0.0f;
+      auto selectionBounds = gfx::RectF();
+      auto textBounds = gfx::RectF();
+      std::map<int, gfx::RectF> glyphsBounds;
```

Comment:
Why a std::map and not a std::vector here? It looks like this variable is being used as a vector/array.

Reply:
You are right, I think I thought that I'd need random access to the elements...I'll change it. Thanks!

Actually that is possible with a vector too...so don't know what I was thinking...